overlord/devicestate/firstboot.go

```diff
@@ -327,7 +329,7 @@ func populateStateFromSeedImpl(st *state.State, opts *populateStateFromSeedOptio
 	return tsAll, nil
 }
 
-func importAssertionsFromSeed(st *state.State, sysLabel string) (seed.Seed, error) {
+func importAssertionsFromSeed(st *state.State, sysLabel string, hasModeEnv bool) (seed.Seed, error) {
```

Comment:
my preference would be to make the extra bool we pass to be isCoreBoot

Reply:
Changed